### PR TITLE
feat!: upgrade to stable nuxt and server dependencies

### PR DESCRIPTION
**Pull request by danielroe**
### 🔗 Linked issue

closes https://github.com/nuxt/bridge/pull/597 (by incorporating), resolves https://github.com/nuxt/bridge/issues/649

### ❓ Type of change

- [x] 🧹 Chore
- [ ] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [x] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

This is a work-in-progress PR to upgrade other nuxt/framework dependencies to v3 as well as upgrading server deps (h3/framework). It includes the refactor in https://github.com/nuxt/bridge/pull/597 that creates a new package (`@nuxt/bridge-schema`) including Nuxt 2 specific types which are now omitted from `@nuxt/schema` following https://github.com/nuxt/framework/pull/8487.

These come with a host of breaking changes which I'll do my best to document before merging this PR.

cc: @pi0 

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: packages/bridge-schema/package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  "scripts": {
+    "build": "unbuild",
+    "prepack": "unbuild"
+  },
+  "devDependencies": {
+    "@types/lodash.template": "^4",
+    "@types/semver": "^7",
+    "@vitejs/plugin-vue": "^3.1.2",
+    "@vueuse/head": "~1.0.0-rc.9",
+    "nitropack": "^0.6.0",
```
Comment (pi0) [inline packages/bridge-schema/package.json:22]: I think we can update nitropack and other deps here as well? (in other package.json it is updated)

File: scripts/release-edge.sh [hunk 2]
```diff
@@ -19,6 +19,9 @@ if [[ ! -z ${NODE_AUTH_TOKEN} ]] ; then
 fi
 
 # Release packages
-echo "Publishing package..."
-pnpm publish --access public --no-git-checks
-
+for p in packages/* ; do
+  pushd $p
+  echo "Publishing $p"
+  pnpm publish --access public --no-git-checks
```
Comment (pi0) [inline scripts/release-edge.sh:25]: Note: We need a dummy publish before merging this PR to add bot

File: packages/bridge/src/nitro.ts [hunk 3]
```diff
@@ -232,7 +232,8 @@ export async function setupNitroBridge () {
       // We use only object form of alias in base config
       if (Array.isArray(config.resolve.alias)) { return }
       config.resolve.alias.ufo = 'ufo/dist/index.mjs'
-      config.resolve.alias.ohmyfetch = 'ohmyfetch/dist/index.mjs'
```
Comment (pi0) [inline packages/bridge/src/nitro.ts:235]: It wouldn't be a bad idea to keep this for any packages still using old name. I also plan to create a placeholder stub releases for `ohmyfetch`

**Review by pi0 — APPROVED** [review]
Other than comments, relly nice work 🚀

File: scripts/release-edge.sh (hunk 2 above)
Comment (pi0) [reply scripts/release-edge.sh:25]: Package is ready https://www.npmjs.com/package/@nuxt/bridge-schema-edge

File: packages/bridge/src/nitro.ts (hunk 3 above)
Comment (danielroe) [reply packages/bridge/src/nitro.ts:235]: oops, that was what the second line was meant to be.

File: packages/bridge-schema/package.json (hunk 1 above)
Comment (danielroe) [reply packages/bridge-schema/package.json:22]: thanks!

**Pull request merged**